From 598bfccd5d5bf637be37068c5ef13600733324d6 Mon Sep 17 00:00:00 2001
From: Shuguang Gong <shuguang.gong@intel.com>
Date: Tue, 19 May 2015 12:34:58 +0800
Subject: [PATCH 401/462] ov2685: correct awb settings

This patch corrects awb settings of fluorescent and incandescent

Change-Id: I47546567815c6c41e1c91cf84fa4ed9a78bb8659
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-35470
Signed-off-by: Shuguang Gong <shuguang.gong@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/56510
Build: IMIN (SC)
Validation-Android: Wei Tao <wei.tao@intel.com>
Approver: Jing Tao <jing.tao@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../external_drivers/camera/drivers/media/i2c/ov2685.h   | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov2685.h b/drivers/external_drivers/camera/drivers/media/i2c/ov2685.h
index 045176c..198df8e 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov2685.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov2685.h
@@ -365,12 +365,12 @@ static struct ov2685_reg const ov2685_AWB_sunny[] = {
 static struct ov2685_reg const ov2685_AWB_incandescent[] = {
 	{OV2685_8BIT, 0x3208 , 0x00},
 	{OV2685_8BIT, 0x5180 , 0xf6},
-	{OV2685_8BIT, 0x5195 , 0x06},
-	{OV2685_8BIT, 0x5196 , 0xb8},
+	{OV2685_8BIT, 0x5195 , 0x04},
+	{OV2685_8BIT, 0x5196 , 0x90},
 	{OV2685_8BIT, 0x5197 , 0x04},
 	{OV2685_8BIT, 0x5198 , 0x00},
-	{OV2685_8BIT, 0x5199 , 0x06},
-	{OV2685_8BIT, 0x519a , 0x5f},
+	{OV2685_8BIT, 0x5199 , 0x09},
+	{OV2685_8BIT, 0x519a , 0x20},
 	{OV2685_8BIT, 0x3208 , 0x10},
 	{OV2685_8BIT, 0x3208 , 0xa0},
 	{OV2685_TOK_TERM, 0, 0}
@@ -379,12 +379,12 @@ static struct ov2685_reg const ov2685_AWB_incandescent[] = {
 static struct ov2685_reg const ov2685_AWB_fluorescent[] = {
 	{OV2685_8BIT, 0x3208 , 0x00},
 	{OV2685_8BIT, 0x5180 , 0xf6},
-	{OV2685_8BIT, 0x5195 , 0x04},
-	{OV2685_8BIT, 0x5196 , 0x90},
+	{OV2685_8BIT, 0x5195 , 0x06},
+	{OV2685_8BIT, 0x5196 , 0xb8},
 	{OV2685_8BIT, 0x5197 , 0x04},
 	{OV2685_8BIT, 0x5198 , 0x00},
-	{OV2685_8BIT, 0x5199 , 0x09},
-	{OV2685_8BIT, 0x519a , 0x20},
+	{OV2685_8BIT, 0x5199 , 0x06},
+	{OV2685_8BIT, 0x519a , 0x5f},
 	{OV2685_8BIT, 0x3208 , 0x10},
 	{OV2685_8BIT, 0x3208 , 0xa0},
 	{OV2685_TOK_TERM, 0, 0}
-- 
1.9.1

